src/components/rockets/ProfileRocket.js

```diff
@@ -0,0 +1,30 @@
+import { useSelector } from 'react-redux';
+import './Rocket.css';
+import React from 'react';
+import ProfileRocketItem from './ProfileRocketItem';
+
+const ProfileRocket = () => {
+  let status = null;
+  const rocket = useSelector((state) => state.Rockets);
+  const filteredRocket = rocket.filter((rocket) => rocket.reserved === true);
+  if (!filteredRocket.length) { status = 'No Rockets reserved'; }
+  return (
+    <>
+      <div className="profile-rocket-container">
+        <h2>My Rockets</h2>
+        <ul className="profile-rocket-list-conatiner">
```

Comment:
- [ ] everything looks fine. Please add a little margin bottom/top to the images, so they can easily be differentiated.
![Screenshot (485)_LI](https://user-images.githubusercontent.com/65298048/193292813-6498f591-443f-4fca-b30b-55df947bf3af.jpg)